### ISSUE TITLE
"\r\n" causing render issues in error messages on Windows

### ISSUE THREAD

**Issue by zwilias** [elm/compiler#1900]
**Quick Summary:** Errors on Windows appear to be offset when a multiline string occurs before them

Note: This only appears to occur on Windows, I cannot reproduce this on OS X. It's a corollary to #1819

## SSCCE

```elm
multiline : String
multiline = """
hello
  """

fail : List String -> List String
fail xs = 1 :: xs
```

On **windows**, this yields:

```
C:\Users\Ilias Van Peer\test>elm make src/Main.elm
Detected errors in 1 module.
-- TYPE MISMATCH -------------------------------------------------- src/Main.elm

Something is off with the body of the `fail` definition:

elm.exe: Prelude.last: empty list
elm.exe: thread blocked indefinitely in an MVar operation
```

If we modify the SSCCE by adding another (correct) definition after the wrong one, like so:

```elm
multiline : String
multiline = """
hello
  """

fail : List String -> List String
fail xs = 1 :: xs

ok = 1
```

We get the following error, where the "position indicator" is misaligned:

```
C:\Users\Ilias Van Peer\test>elm make src/Main.elm
Detected errors in 1 module.
-- TYPE MISMATCH -------------------------------------------------- src/Main.elm

Something is off with the body of the `fail` definition:

9| ok = 1
             ^^^^^^^
The body is:

    List number

But the type annotation on `fail` says it should be:

    List String

Hint: Try using String.fromInt to convert it to a string?

-- TYPE MISMATCH -------------------------------------------------- src/Main.elm

I am having trouble with this (::) operator:

9| ok = 1
             ^^^^^^^
The left side of (::) is:

    number

But you are trying to put that into a list filled with:

    String

Lists need ALL elements to be the same type though.

Hint: Try using String.fromInt to convert it to a string?
```

- **Elm:** 0.19.0
- **Browser:** n/a
- **Operating System:** Windows 10

**Comment by Kurren123** [comment]
Can confirm, happening on windows 10.

**Comment by evancz** [comment]
Just tried this with the master branch on Windows, and the first example gives the following error:

```
-- TYPE MISMATCH ------------------------------------------------------ temp.elm

Something is off with the body of the `fail` definition:

7| fail xs = 1 :: xs
             ^^^^^^^
The body is:

    List number

But the type annotation on `fail` says it should be:

    List String

Hint: Try using String.fromInt to convert it to a string?
```

So it looks like the root issue was resolved along with #1819. The fix will become available when 0.19.1 is released.
